src/main/java/alfio/manager/payment/BaseStripeManager.java

```diff
@@ -113,7 +113,7 @@ StripeConnectURL getConnectURL(Function<ConfigurationKeys, Configuration.Configu
     }
 
     private boolean isConnectEnabled(PaymentContext context) {
-        return configurationManager.getBooleanConfigValue(context.narrow(PLATFORM_MODE_ENABLED), false);
+        return configurationManager.getFor(context.getEvent(), PLATFORM_MODE_ENABLED).getValueAsBooleanOrDefault(false);
```

Comment:
This is not safe, as it could cause a NPE. See https://github.com/alfio-event/alf.io/blob/refactor-configuration-get-value/src/main/java/alfio/model/transaction/PaymentContext.java#L34

Reply:
fixed in the new commit, if event is null, it will fetch the configuration at system level